podigee/job/__main__.py

```diff
@@ -269,14 +350,30 @@ def transform_podigee_analytics_to_metrics(analytics_data):
             openpodcast_endpoint="metrics",
             podigee_call=get_request_lambda(
                 lambda ep_id: transform_podigee_analytics_to_metrics(
-                    podigee.episode_analytics(ep_id, granularity=None, start=date_range.start, end=date_range.end)
-                ), 
+                    podigee.episode_analytics(ep_id, granularity=None, start=date_range.start, end=date_range.end),
+                    # for now we just store the downloads and do not store platforms etc. per episode
+                    downloads_only=True
```

Comment:
Can we rename that to `store_downloads_only`? I got confused for a while because the rest of the data is clearly fetched, just not stored.